packages/errors/__snapshot-html__/CDP_FIREFOX_DEPRECATED.html

```diff
@@ -34,7 +34,5 @@
     </style>
   
     </head>
-    <body><pre><span style="color:#e05561">You&#39;re not logged in.<span style="color:#e6e6e6">
-<span style="color:#e05561"><span style="color:#e6e6e6">
-<span style="color:#e05561">Run <span style="color:#e5e510">cypress open<span style="color:#e05561"> to open Cypress and log in.<span style="color:#e6e6e6"></span></span></span></span></span></span></span></span>
+    <body><pre><span style="color:#e05561">Since Firefox 129, Chrome DevTools Protocol (CDP) has been deprecated in Firefox. In Firefox 135 and above, Cypress defaults to automating the Firefox browser with WebDriver BiDi. Cypress will no longer support CDP within Firefox in the future and is planned for removal in Cypress 15.<span style="color:#e6e6e6"></span></span>
```

Comment:
This is weird that git identifies this as renaming this file. This is removing the NOT_LOGGED_IN snapshot, but I've also noticed running the errors unit tests a few times that we forgot to commit the CDP_FIREFOX_DEPRECATED snapshot - so that's being done in this PR also.

Reply:
thank you for doing that!